Minor improvements to zig install scripts

brson opened the pull request:
Three minor changes:

- Set the mode on install_zig.bat to executable. This makes it executable by cygwin.
- Modify install_zig.sh to work with cygwin.
- Modify install_zig.sh not to display stray "/usr/bin/wget" output when probing for wget.

I have not tested on macos, but it still looks correct to me.

brson committed:
Make download.sh compatible with cygwin

brson committed:
download.sh: Don't print results of wget command probe

This prints "/usr/bin/wget" to the terminal,
which is unnecessary and ugly.
Send it to /dev/null instead.

jorangreef commented:
@brson my apologies! I only saw your work here after conflicting nicely with #2180, would you please layer on after that lands?

brson commented:
I've rebased this and retested. I removed the commit to set the executable bit on download.bat, as I was finding it confusing on linux - autocomplete was suggesting it, and executing it would fail. With install.sh being cygwin compatible, that is a reasonable choice for running under cygwin anyway.
